package.json

```diff
@@ -8,11 +8,11 @@
         "lib"
     ],
     "dependencies": {
-        "@hapi/pinpoint": "1.x.x"
+        "@hapi/pinpoint": "2.x.x"
```

Comment:
This module has already moved out of the hapi org, but a new version hasn't been published yet.

Reply:
Thx for letting me know! You think we should just wait on this one then?

Reply:
@hueniverse are you planning to publish a new pinpoint any time soon?

Reply:
Not soon. This is a test module so you can just leave as-is.